.github/workflows/tests-docker.yml

```diff
@@ -26,35 +26,29 @@ jobs:
         cp tests/test.env dev.env
         cat dev.env
         python3 wis2box-ctl.py config
-    - name: install loki plugin for docker
```

Comment:
Don't we still need the plugin install?

Reply:
This happens in `python3 wis2box-ctl.py start`

Reply:
+1